python/ray/data/dataset.py

```diff
@@ -2966,6 +2967,8 @@ def write_parquet(
         Args:
             path: The path to the destination root directory, where
                 parquet files are written to.
+            partition_cols: Column names by which to partition the dataset.
+                Files are writted in Hive partition style.
```

Comment:
Nit: Use active voice (from our style guide: https://developers.google.com/style/voice)

Also, typo with "writted" 
```suggestion
            partition_cols: Column names by which to partition the dataset.
                This methods writes files in Hive partition style.
```